### PR TITLE
Clean up `rake gateways:hosts` output

### PR DESCRIPTION
**Pull request by ntalbott**
@Shopify/active-merchant this look good to close out #1478?

### CODE REVIEW

**Comment by girasquid** [comment]
This looks good - why the remote ePay test changes?

File: lib/support/outbound_hosts.rb [hunk 1]
```diff
@@ -2,24 +2,27 @@
...
     end
 
-    uris.each do |uri|
-      puts "#{uri.first} #{uri.last}"
-    end
+    [hosts, invalid_lines]
   end
-end
\ No newline at end of file
+end
```
Comment (j-mutter) [inline lib/support/outbound_hosts.rb]: :heart_eyes:

File: lib/active_merchant/billing/gateways/epay.rb [hunk 2]
```diff
@@ -175,20 +174,21 @@ def messages(epay, pbs = nil)
...
 
-        response = raw_ssl_request(:post, 'https://' + API_HOST + '/auth/default.aspx', authorize_post_data(params), headers)
+        response = raw_ssl_request(:post, live_url + 'auth/default.aspx', authorize_post_data(params), headers)
 
         # Authorize gives the response back by redirecting with the values in
         # the URL query
         if location = response['Location']
           query = CGI::parse(URI.parse(location.gsub(' ', '%20')).query)
         else
+          p response.body
```
Comment (j-mutter) [inline lib/active_merchant/billing/gateways/epay.rb]: Accidental?

File: test/remote/gateways/remote_epay_test.rb [hunk 3]
```diff
@@ -10,96 +10,63 @@ def setup
     @credit_card_declined = credit_card('3333333333333102')
 
     @amount = 100
-    @options = { :order_id => '1' }
+    @options = {order_id: '1'}
```
Comment (j-mutter) [inline test/remote/gateways/remote_epay_test.rb]: :+1:

**Comment by ntalbott** [comment]
@girasquid cause I was trying to run them due to URL changes, and took the opportunity to clean them up.

File: lib/active_merchant/billing/gateways/epay.rb (hunk 2 above)
Comment (ntalbott) [inline lib/active_merchant/billing/gateways/epay.rb]: Yup. Will fix.

**Comment by girasquid** [comment]
Oh ok - :cool:

**Pull request closed without merging**